server/modules/protocol/MySQL/mysql_common.cc

```diff
@@ -1768,3 +1767,126 @@ void mxs_mysql_execute_kill_user(MXS_SESSION* issuer, const char* user, kill_typ
 
     mxs_mysql_send_ok(issuer->client_dcb, info.targets.size(), 0, NULL);
 }
+
+/**
+ *  Decode ok packet and get session track info
+ *
+ *  @param buffer               Buffer contain ok packet
+ *  @param server_capabilities  Server capabilities
+ *
+ */
+void mxs_mysql_get_session_track_info(GWBUF *buff, uint32_t server_capabilities)
+{
+    char *trx_info, *var_name, *var_value;
+
+    if (GWBUF_LENGTH(buff) < 9  || !mxs_mysql_is_ok_packet(buff))
+    {
+        return;
+    }
+
+    if (!GWBUF_IS_CONTIGUOUS(buff))
+    {
+        buff = gwbuf_make_contiguous(buff);
+    }
+
+    buff->gwbuf_type |= GWBUF_TYPE_REPLY_OK;
+
+    uint8_t* ptr = GWBUF_DATA(buff);
+
+    ptr += (MYSQL_COM_OFFSET + 1); // Header and Command type
```

Comment:
Use the `MYSQL_HEADER_LEN` macro from `<maxscale/protocol/mysql.h>`, this is what is used elsewhere.